webapp/pages/group/_id/_slug.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-if="isGroupVisible">
+  <div class="group-profile" v-if="isGroupVisible">
```

Comment:
What does this have to do with the branding changes?

Reply:
It enables us to change the `ds-chip` background color in yunite just of this page.